platform/jobs/edxPlatformAccessibilityPr.groovy

```diff
@@ -57,18 +57,8 @@ Map privateJobConfig = [
     workerLabel: 'js-worker',
     whitelistBranchRegex: /^((?!open-release\/).)*$/,
     context: 'jenkins/a11y',
-    triggerPhrase: /.*jenkins\W+run\W+a11y.*/
-]
-
-Map python3JobConfig = [
-    open : true,
-    jobName : 'edx-platform-python3-accessibility-pr',
-    repoName : 'edx-platform',
-    workerLabel: 'js-worker',
-    whitelistBranchRegex: /^((?!open-release\/).)*$/,
-    context: 'jenkins/python3.5/a11y',
-    triggerPhrase: /.*jenkins\W+run\W+py35-django111\W+a11y.*/,
-    toxEnv: 'py35-django111'
```

Comment:
^ i'm out of the loop. Have we stopped using tox?

Reply:
We were using tox for the Python 3 jobs because we modeled them after the Django upgrade jobs where we had to use it.  But they should work fine by just using a Python 3 virtualenv, also.  We'll start using tox again for optional Django 2.0, 2.1, and 2.2 jobs shortly.

Reply:
oh, right. Thanks